src/telemetry/metrics.yaml

```diff
@@ -100,6 +103,105 @@ impression:
     expires: never
     extra_keys: *impression_extra_keys
 
+  signup_screen_view:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      The user has just moved on to the screen that has the
+      "Sign up" button
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-5251
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/8060#issuecomment-1724272061
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - brizental@mozilla.com
+      - vpn-telemetry@mozilla.com
+    expires: never
+    extra_keys: *impression_extra_keys
+
+  enter_email_view:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      The user has just moved on to the sign up screen
+      where they are prompted for their email.
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-5251
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/8060#issuecomment-1724272061
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - brizental@mozilla.com
+      - vpn-telemetry@mozilla.com
+    expires: never
+    extra_keys: *impression_extra_keys
+
+  authentication_inapp_step:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      The user has moved on to a new screen in the authentication flow.
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-5251
+    data_reviews:
+      - TODO
```

Comment:
I'm not fully aware of how telemetry logistics work but is this going to be changed before the PR is merged?

Reply:
Yes, but it's awaiting on other discussions. https://github.com/mozilla-mobile/mozilla-vpn-client/pull/8139#issuecomment-1755563514